sponsorship_compassion/models/res_partner.py

```diff
@@ -416,21 +416,56 @@ def _random_str():
                 "category_id": [(5, 0, 0)],
                 "comment": False,
                 "active": False,
+                "city": False,
+                "zip": False,
+                "commercial_company_name": False,
+                "company_name": False,
+                "contact_address": False,
+                "display_name": False,
+                "email_formatted": False,
+                "email_normalized": False,
+                "gmc_gender": False,
+                "lastname": _random_str(),
+                "phone_sanitized": False,
+                "salutation": False,
```

Comment:
You can remove this field, because it's a computed field (not stored in database, and depends on the other fields you delete).